css/colorpicker.css

```diff
@@ -1,8 +1,3 @@
-.colorpicker input{
-	width: auto;
-	outline-color: none;
-}
-
 .colorpicker input:focus{
 	outline: 0;
 }

```

Comment:
Could you move these 3 lines right after Line 94? So that CSS rules about `.colorpicker input` can be grouped closer to each other.

Reply:
Done